demos/link-prediction/homogeneous-comparison-link-prediction.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>It's interesting to me that all of these algorithms are all very similar to the base Node2Vec algorithm.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Atti2Vec is better, while GraphSAGE and GCN has comparable performance with Node2Vec. Graph convolution or neighbourhood aggregation benefits the node classification task, which largely depends on node attributes for the Cora dataset. In comparison, it is hard for them to boost the link prediction performance, which is dominated by the structure information.</p>